apps/prairielearn/src/lib/load.ts

```diff
@@ -42,7 +42,7 @@ class LoadEstimator {
   startJob(id: string) {
     debug(`LoadEstimator.startJob(): jobType = ${this.jobType}, id = ${id}`);
     this._addIntegratedLoad();
-    if (_.has(this.currentJobs, id)) {
+    if (Object.prototype.hasOwnProperty.call(this.currentJobs, id)) {
```

Comment:
Since currentJobs is just a record, would `id in this.currentJobs` be easier to read? Also in other places.

Reply:
`id` could (in theory) itself be something like `"hasOwnProperty"`. I'm fine using `in` directly when the property is a static string. If it's dynamic, as it is here, I used the method you see here.